Dockerfile

```diff
@@ -17,5 +17,7 @@ RUN cd /opt/oracle/instantclient_12_2 \
 RUN echo /opt/oracle/instantclient_12_2 > /etc/ld.so.conf.d/oracle-instantclient.conf \
  && ldconfig
 
+RUN gulp test && rm -rf logs
```

Comment:
Is there a way to disable logging in the first place? It seems weird to log to a directory and then immediately delete it. We might want to view those logs at some point, maybe?